pkg/cmd/cli/install/install.go

```diff
@@ -179,11 +179,27 @@ func GetVeleroVersion(f client.Factory) (string, error) {
 // If currentVersion < minVersion, return -1
 // Assume input string format is vX.Y.Z
 func CompareVersion(currentVersion string, minVersion string) int {
+	if currentVersion == "" {
+		currentVersion = "v0.0.0"
+	}
+	if minVersion == "" {
+		minVersion = "v0.0.0"
+	}
 	curVerArr := strings.Split(currentVersion[1:], ".")
 	minVerArr := strings.Split(minVersion[1:], ".")
+
 	for index, _ := range curVerArr {
```

Comment:
If len(curVerArr) < len(minVerArr) there are some issue, e.g curVerArr:v1.3.0, minVerArr:v1.3.0.1, this expect to return -1, but will return 0 in this code. Let's use the longer string to run the loop and change this to:
var length int
if len(curVerArr) < len(minVerArr) {
    length = len(minVerArr)
} else {
    length = len(curVerArr)
}
index := 0
for ; index < length; index++ {
    // inside compare code
}